js/src/api/util/types.js

```diff
@@ -29,6 +29,10 @@ export function isFunction (test) {
 }
 
 export function isHex (_test) {
+  if (!isString(_test)) {
```

Comment:
+1 Was just wondering about this. (Apart from the use-case, makes it more robust overall)